content/learn/vp_agof_config_controller.adoc

```diff
@@ -0,0 +1,223 @@
+    scm_url: 'https://github.com/hybrid-cloud-demos/agof_minimal_demo.git'
+
+controller_organizations:
+  - name: '{{ orgname_vault }}'
+
+controller_inventories:
+  - name: 'AGOF Demo Inventory'
+    organization: '{{ orgname_vault }}'
+
+controller_inventory_sources:
+  - name: 'AGOF Demo Inventory Source'
+    inventory: 'AGOF Demo Inventory'
+    credential: 'ec2_ssh_credential'
+    overwrite: true
+    overwrite_vars: true
+    update_on_launch: true
+    source: scm
+    source_project: '{{ agof_demo_project_name }}'
+    source_path: 'inventory'
+
+controller_credential_types: []
+
+controller_templates:
+  - name: Demo Job Template
+    state: absent
+
+  - name: Ping Playbook
+    organization: "{{ orgname_vault }}"
+    project: '{{ agof_demo_project_name }}'
+    job_type: run
+    playbook: 'ansible/playbooks/ping.yml'
+    inventory: "AGOF Demo Inventory"
+    credentials:
+      - ec2_ssh_credential
+
+controller_schedules:
+  - name: Ping Playbook
+    organization: "{{ orgname_vault }}"
+    unified_job_template: Ping Playbook
+    rrule: DTSTART:20191219T130500Z RRULE:FREQ=MINUTELY;INTERVAL=120
+
+demo_ssh_key_file: '~/{{ ec2_name_prefix }}/{{ ec2_name_prefix }}-private.pem'
+
+controller_credentials:
+  - name: ec2_ssh_credential
+    description: "EC2 SSH credential"
+    organization: '{{ orgname_vault }}'
+    credential_type: Machine
+    inputs:
+      username: 'ec2-user'
+      ssh_key_data: "{{ lookup('file', demo_ssh_key_file) }}"
+      become_method: sudo
+
+controller_launch_jobs:
+  - name: Ping Playbook
+    organization: "{{ orgname_vault }}"
+----
+
+This file automates the creation, updating, or deletion of Ansible Controller objects (organizations, projects, inventories, credentials, templates, schedules). Sensitive information like passwords and keys are pulled dynamically from vaults, ensuring they are not hardcoded in the configuration. 
+
+A a Git repository manages the project’s inventory and playbooks, allowing for continuous integration and delivery (CI/CD) practices. AAP automatically schedules recurring playbook executions, eliminating the need for manual job triggers.
+
+== Key sections and parameters 
+
+This section describes the parameters associated with the Ansible GitOps Framework minimal configuration demo. 
+
+=== Vault variables
+
+*`orgname_vault: 'Demo Organization'`*:: 
+This specifies the organization name stored in a vault for security purposes.
+
+*`controller_username_vault: 'admin'`*:: 
+This is the Ansible Controller's username stored in a vault.
+
+*`controller_password_vault: '{{ admin_password }}'`*:: 
+The password is fetched dynamically from a vault for security purposes.
+
+=== Dynamic variables
+
+*`controller_username: '{{ controller_username_vault }}'`*:: 
+The Ansible Controller username is retrieved from the vault variable.
+
+*`controller_password: '{{ controller_password_vault }}'`*:: 
+The password is dynamically fetched from the vault.
```

Comment:
The _vault convention is used in the Ansible ecosystem to indicate that the actual variable is stored in a vault (file). This allows a bit of loose coupling between the variable names and their actual usage in Ansible